server/chat-commands/moderation.ts

```diff
@@ -1534,15 +1586,16 @@ exports.commands = {
 			this.errorReply(`- other: get a leader`);
 			return;
 		}
-		if (Punishments.isBattleBanned(targetUser)) return this.errorReply(`User '${targetUser.name}' is already banned from battling.`);
-
+		if (Punishments.isBattleBanned(targetUser)) {
+			return this.errorReply(`User '${targetUser.name}' is already banned from battling.`);
+		}
 		const reasonText = reason ? ` (${reason})` : `.`;
 		const battlebanMessage = `${targetUser.name} was banned from starting new battles by ${user.name}${reasonText}`;
 		this.privateModAction(`(${battlebanMessage})`);
 
 		// Notify staff room when a user is banned from battling outside of it.
 		if (room.roomid !== 'staff' && Rooms.get('staff')) {
```

Comment:
```suggestion
		if (room.roomid !== 'staff') {
```

Do you understand why I'm doing this? If you change `Rooms.get('staff').` to `Rooms.get('staff')?.`, this check becomes redundant.

You should never change `.` to `?.` if you don't understand why it wasn't like that before.